packages/block-library/src/comment-author-avatar/index.php

```diff
@@ -25,6 +25,23 @@ function render_block_core_comment_author_avatar( $attributes, $content, $block
 
 	// This is the only way to retreive style and classes on different instances.
 	$wrapper_attributes = WP_Block_Supports::get_instance()->apply_block_supports();
+
+	/**
+	 * We get the spacing attributes and transform the array provided into a string formatted for being applied as a style html tag.
+	 * Good candidate to be moved to a separate function in core.
+	 *
+	**/
+	$spacing_attributes = isset($attributes['style']['spacing']) ? $attributes['style']['spacing'] : null;
+	if ( isset($spacing_attributes) && ! empty( $spacing_attributes ) ) {
```

Comment:
There are formatting issues detected by CI:

https://github.com/WordPress/gutenberg/runs/4143515671?check_suite_focus=true#step:6:173

`npm run format-php` should fix code formatting.

Reply:
Couldn't do it before as I was having weird docker errors. Fixed!